pkgs/build-support/build-pecl.nix

```diff
@@ -23,4 +23,8 @@ stdenv.mkDerivation (args // {
   makeFlags = [ "EXTENSION_DIR=$(out)/lib/php/extensions" ] ++ makeFlags;
 
   autoreconfPhase = "phpize";
+
+  preConfigure = ''
```

Comment:
Is this step really needed in *all* pecl builds or just a single one? If it's a single one it shouldn't be added globally.

Reply:
Required to all pecl builds. Checked through nix log.